Rosieks.VisualStudio.Angular/Commands/GoToCodeCmd.cs

```diff
@@ -92,9 +94,14 @@ private void MenuItemCallback(object sender, EventArgs e)
         private string GetCodePath(string path)
         {
             var fileName = Path.GetFileNameWithoutExtension(path);
-            fileName = fileName.Contains(".directive.") ? fileName : fileName + ".controller";
-            string codePath = Path.Combine(Path.GetDirectoryName(path), fileName);
-            if (FileHelper.TryFind(codePath, AngularPackage.CodeExtensions, out codePath))
+            var searchPaths = new List<string>();
+
+            var adjustedFileName = fileName.Contains(".directive.") ? fileName : fileName + ".controller";
```

Comment:
If the original `fileName` is for example _menu.directive.js_ then you will search for such files:
* `adjustedFileName` -> _menu.directive.js_
* `fileName` -> _menu.directive.js_

And as I understood you would like to look for _menu.js_